.github/workflows/check-npm-version.yml

```diff
@@ -7,18 +7,35 @@ on:
 jobs:
   start:
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Check if version has been updated
+      - name: Check if package version has been updated
         id: check
         uses: EndBug/version-check@v2.1.1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           file-name: ./package.json
-      - name: Log when changed
+      - name: Verify Changelog changes
+        uses: tj-actions/verify-changed-files@v14
+        id: verify-changed-files
+        with:
+          files: |
+            CHANGELOG.md
+      - name: Log when package version changed
         if: steps.check.outputs.changed == 'true'
         run: 'echo "Yayy!! Version change found in commit ${{ steps.check.outputs.commit }}! New version: ${{ steps.check.outputs.version }} (${{ steps.check.outputs.type }})"'
-      - name: Log when unchanged
+      - name: Log when package version unchanged
         if: steps.check.outputs.changed == 'false'
         run: echo "No version change :/ Please update version in package.json!" && exit 1
```

Comment:
Could we also have this check add a PR comment?

Reply:
Thanks for the suggestion! Made the changes and the comment is shown below